zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/CommitProcessor.java

```diff
@@ -213,18 +244,21 @@ public void run() {
                  * is present along with a pending local write. After the loop,
                  * we process one committed request if commitIsWaiting.
```

Comment:
I feel this comment should be updated:
* a committed request will not stop the request processing if maxReadBatchSize is set.
* commits might be processed in batch instead of one commit, after the loop.